www/src/Locales/en/Addons/PSX.jsx

```diff
@@ -0,0 +1,9 @@
+export default {
+	'header': 'PSX',
+	'text': 'blablabla',
```

Comment:
This text should have something closer to what SNES.jsx translation looks like. Something like this add-on supports Playstation controller input for X controllers.